prismic-model/js/parts/body.js

```diff
@@ -99,6 +99,11 @@ export default {
           },
         },
       }),
+      venueTimes: slice('Venue opening times', {
```

Comment:
Or could this be something more generic like `content-slice`? 
Have we had any thoughts about bringing `collection-venue` and `places` together?